addon/mixins/route.js

```diff
@@ -301,6 +301,7 @@ const RouteMixin = Mixin.create({
   _doUpdate(queryObject, infinityModel) {
     const totalPages = queryObject.get(get(infinityModel, 'totalPagesParam'));
     set(infinityModel, '_totalPages', totalPages);
+    set(infinityModel, 'meta', get(queryObject, 'meta'));
```

Comment:
Perhaps add a test here? https://github.com/hhff/ember-infinity/blob/1-0/tests/unit/mixins/route-test.js#L195

`assert.equal(model.get('meta'), { total_pages: 31 }, 'meta');`

Reply:
And also add the documentation here: https://github.com/hhff/ember-infinity/blob/1-0/addon/lib/infinity-model.js#L78

Reply:
Boom! Done.